qa/automation/CareerPortal.Tests.Scenarios/4. Send CV/4.1. Send CV.feature

```diff
@@ -0,0 +1,28 @@
+Feature: 4.1 send CV
+Background:
+	Given User opened Career Portal
+	
+
+Scenario Outline: 4.1_1 Send CV
```

Comment:
Can't get the point of the test by the title. Suppose better to specify, what exactly tests in the case, like 'Success send CV  - all entered values are valid' (just my thoughts).